cwltool/__init__.py

```diff
@@ -1 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
I added this when I included ``unicode_literals`` import. I can delete this if we do not plan to use any unicode chars in our source code.